test/vtcomposite-localize-worldview.test.js

```diff
@@ -0,0 +1,352 @@
+'use strict';
+
+const localize = require('../lib/index.js').localize;
+const { vtinfo, getFeatureById } = require('./test-utils.js');
+const mvtFixtures = require('@mapbox/mvt-fixtures');
+const test = require('tape');
+
+const fixtureDefaults = mvtFixtures.create({
+  layers: [
+    {
+      version: 2,
+      name: 'admin',
+      features: [
+        {
+          id: 10,
+          tags: [ 0, 0 ], // _mbx_worldview: US,CN,JP,IN
+          type: 1, // point
+          geometry: [ 9, 54, 38 ]
+        }
+      ],
+      keys: [ '_mbx_worldview' ],
+      values: [
+        { string_value: 'US,CN,JP,IN' }
+      ],
+      extent: 4096
+    }
+  ]
+}).buffer;
+
+const fixtureWithAll = mvtFixtures.create({
+  layers: [
+    {
+      version: 2,
+      name: 'admin',
+      features: [
+        {
+          id: 10,
+          tags: [ 0, 0 ], // _mbx_worldview: all
+          type: 1, // point
+          geometry: [ 9, 54, 38 ]
+        }
+      ],
+      keys: [ '_mbx_worldview' ],
+      values: [
+        { string_value: 'all' }
+      ],
+      extent: 4096
+    }
+  ]
+}).buffer;
+
+test('[localize worldview] defaults - worldview: US specified, only US created', (assert) => {
+  const params = {
+    buffer: fixtureDefaults,
+    worldview: ['US'],
+    // worldview_property: '_mbx_worldview'
+  };
+  localize(params, (err, vtBuffer) => {
+    assert.ifError(err);
+    const tile = vtinfo(vtBuffer);
+    assert.ok('admin' in tile.layers, 'has admin layer');
+    assert.equal(tile.layers.admin.length, 1, 'has one feature');
+    assert.deepEqual(tile.layers.admin.feature(0).properties, {
+      worldview: 'US'
+    }, 'expected properties');
+    assert.end();
+  });
+});
+
+test('[localize worldview] no worldview specified, feature with "all" value is retained', (assert) => {
```

Comment:
what about fixtureDefault and worldview = []. What happens then? Maybe that should get a test

Reply:
https://github.com/mapbox/vtcomposite/pull/123/commits/3de0357cf2203643a05ec4b7fbb08f843dd1e1b3

Added a test with `worldviews: []` to confirm it filters out any properties with a matching `worldview_property` and does not filter out features without it.